From 78809960bbe804e5e413b9af911a37c69a23270f Mon Sep 17 00:00:00 2001
From: Herman Chen <herman.chen@rock-chips.com>
Date: Mon, 5 Sep 2016 17:22:21 +0800
Subject: [PATCH] rockchip/vcodec: remove noisy log on encoder

New user encoder will send larger register size to kernel with extra
info right after the register file. The extra information will specify
the offset patch for address registers. This behavior will cause a lot
of noisy log. So we need to disable it.

Change-Id: I32609be325482b445ee6e97f7f4a245d6d0dc9c2
Signed-off-by: Herman Chen <herman.chen@rock-chips.com>
---
 drivers/video/rockchip/vcodec/vcodec_service.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/video/rockchip/vcodec/vcodec_service.c b/drivers/video/rockchip/vcodec/vcodec_service.c
index d9cf0c6caf98..4c837de2c06a 100644
--- a/drivers/video/rockchip/vcodec/vcodec_service.c
+++ b/drivers/video/rockchip/vcodec/vcodec_service.c
@@ -1211,8 +1211,6 @@ static struct vpu_reg *reg_init(struct vpu_subdev_data *data,
 	}
 
 	if (size > data->reg_size) {
-		pr_err("vpu reg size %u is larger than hw reg size %u\n",
-		       size, data->reg_size);
 		extra_size = size - data->reg_size;
 		size = data->reg_size;
 	}
-- 
2.35.3

